src/__test__/components/plots-and-tables/ViolinPlotIndex.test.jsx

```diff
@@ -135,7 +135,7 @@ describe('ViolinIndex', () => {
   it('allows selection of the grouping/points, defaulting to louvain and All', async () => {
     await renderViolinIndex();
 
-    const dataSelection = rtl.screen.getByText('Select Data');
+    const dataSelection = rtl.screen.getByText('Select data');
```

Comment:
consider using case-insensitive matching to avoid all these issues: 
https://testing-library.com/docs/queries/about#textmatch-examples